src/libs/repository.ts

```diff
@@ -447,3 +449,40 @@ export const requestLifecycleBadgeIfRequired = async (
     throw err;
   }
 };
+
+/**
+ * If a repo has been inactive for certain period,
+ * create an issue reminding it.
+ * @param context The event context context
+ * @param owner The organization name
+ * @param repo The repo name
+ */
+ export const remindInactiveRepository = async (context: Context, owner: string, repo: string) => {
+  try {
+    const updatedAt = context.payload.repository.updated_at;
+
+    const daysInactive = getDaysPassed(updatedAt);
+
+    const isConsidredAsInactive = INACTIVE_DAYS < daysInactive;
```

Comment:
I am not sure if Jason will be picky about this, just a typo in considered for this variable name